pkg/solana/chainwriter/chain_writer.go

```diff
@@ -311,14 +311,15 @@ func (s *SolanaChainWriterService) SubmitTransaction(ctx context.Context, contra
 	// Filter the lookup table addresses based on which accounts are actually used
 	filteredLookupTableMap := s.FilterLookupTableAddresses(accounts, derivedTableMap, staticTableMap)
 
+	options := []txmutils.SetTxConfig{}
 	// Transform args if necessary
 	if methodConfig.ArgsTransform != "" {
 		transformFunc, tfErr := FindTransform(methodConfig.ArgsTransform)
 		if tfErr != nil {
 			return errorWithDebugID(fmt.Errorf("error finding transform function: %w", tfErr), debugID)
 		}
 		s.lggr.Debugw("Applying args transformation", "contract", contractName, "method", method)
-		args, accounts, err = transformFunc(ctx, args, accounts, derivedTableMap)
+		args, accounts, options, err = transformFunc(ctx, args, accounts, derivedTableMap)
```

Comment:
Think we'll need to move this call further up before `GetAddresses` so `ExtraArgsDecoded` is available to resolve the UserAccounts addresses

Reply:
Maybe worth adding an account to one of the test configs for this too?

Reply:
But we need to resolve the account first to pass into the transform func no?

Reply:
Oh duh right. Should we just manually add UserAccounts in args transform also? That way we're just adding all of the remaining accounts in the transform

Reply:
If that's the only way forward, it might make more sense to change that in my PR

Reply:
Don't we have user accounts in the config already? https://github.com/smartcontractkit/chainlink/blob/e97537621be083ea457818a693fb9affc76e3475/core/capabilities/ccip/configs/solana/chain_writer.go#L238-L246 We have to update the URI but this will still work I think?

Reply:
Sorry think I confused myself with something else. You can ignore this.